docs/UserGuide.md

```diff
@@ -130,18 +130,26 @@ Shows a list of all persons in the application.
 Format: `list [s/FIELD]`
 
 * Sorts the contacts by the specified field in **ascending** order. `FIELD` must take one the following values:
-  * `n` sort by name
+  * `n` sort by name ignoring case differences
```

Comment:
I just found out that the allowed name field input is alphanumeric, I am not sure if we need to specify the order between numbers and letters, e.g. the name `123 test` goes  before or after `Alex`

Reply:
Currently the sorting is done lexicographically, which means `123aaron` will come before `aaron` since numbers have a smaller ASCII value than letters. So I don't think this needs to be specified in the UG because this is the standard behaviour when sorting strings 🤔